docs/multi-cluster-ingress.md

```diff
@@ -0,0 +1,303 @@
+# Multi-cluster Ingress
+
+Users can use [MultiClusterIngress API](https://github.com/karmada-io/karmada/blob/master/pkg/apis/networking/v1alpha1/ingress_types.go) provided in Karmada to import external traffic to services in the member clusters.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [quick-start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
+
+### Cluster Network
+
+Currently, we need to use the [MCS](https://github.com/karmada-io/karmada/blob/master/docs/multi-cluster-service.md#the-serviceexport-and-serviceimport-crds-have-been-installed) feature to import external traffic.
+
+So we need to ensure that the container networks between the **host cluster** and member clusters are connected.  The **host cluster** indicates the cluster where the Karmada control plane is deployed.
+
+- If you use the `hack/local-up-karmada.sh` script to deploy Karmada, Karmada will have three member clusters, and the container networks between the **host cluster**, `member1` and `member2` are connected.
+- You can use `Submariner` or other related open source projects to connected networks between clusters.
+
+## Example
+
+### Step 1: Deploy ingress-nginx on the host cluster
+
+We use [multi-cluster-ingress-nginx](https://github.com/karmada-io/multi-cluster-ingress-nginx) as the demo for demonstration. We've made some changes based on the latest version(controller-v1.1.1) of [ingress-nginx](https://github.com/kubernetes/ingress-nginx).
+
+#### Download code
+
+```
+// for HTTPS
+git clone https://github.com/karmada-io/multi-cluster-ingress-nginx.git
+// for SSH
+git clone git@github.com:karmada-io/multi-cluster-ingress-nginx.git
+```
+
+#### Build and deploy ingress-nginx
+
+Using existing `karmada-host` kind cluster to building and deploying the ingress controller.
```

Comment:
Use the existing `karmada-host` kind cluster to build and deploy the ingress controller.